### PR TITLE
Adapt workflow example to support PRs from forks

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/129730

### CODE REVIEW

File: .github/workflows/openqa.yml [hunk 1]
```diff
@@ -3,12 +3,17 @@ name: openQA - GitHub action example
 # yamllint disable-line rule:truthy
 on:
   # triggering on PRs is disabled here in favor of using the webhook-based approach
-  #pull_request:
+  # note: Use `pull_request_target` and not `pull_request` to enabled it. Only then
+  #       the workflow runs within the context of the main repository. This is
+  #       necessary to allow accessing the secrets for API access.
+  #pull_request_target:
```
Comment (kalikiana) [inline .github/workflows/openqa.yml:9]: So "adapt" means switch off? I'm not clear what you want to achieve here.

Comment (Martchus) [reply .github/workflows/openqa.yml:9]: It has already been switched off before. This PR is not changing that. I'm just changing the already disabled example code here so it is covering the use case of creating a PR from a fork repository.

**Review by perlpunk — APPROVED** [review]

File: .github/workflows/openqa.yml [hunk 2]
```diff
@@ -3,12 +3,17 @@ name: openQA - GitHub action example
 # yamllint disable-line rule:truthy
 on:
   # triggering on PRs is disabled here in favor of using the webhook-based approach
-  #pull_request:
+  # note: Use `pull_request_target` and not `pull_request` to enabled it. Only then
```
Comment (okurz) [inline .github/workflows/openqa.yml:6]: s/enabled/enable/

**Review by okurz — COMMENTED** [review]
minor typo. Resolve that, then auto-approved and you can merge

**Comment by Martchus** [comment]
Looks like this is actually in conflict and this change has already been done - with the check enabled, though. Unfortunately it currently fails with `Reason: isotovideo died: Unable to clone Git repository 'https://github.com/${github.event.pull_request.head.repo.full_name}}.git#ci-3' specified via CASEDIR (see log for details) at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 55.` (see https://openqa.opensuse.org/tests/3355483). So I guess the variable substitution doesn't work after all. @mimi1vx Any ideas?

**Comment by mimi1vx** [comment]
typo in my PR ... I'll send fix

**Comment by Martchus** [comment]
Thanks!

**Pull request closed without merging**